src/components/atoms/mainHeading/MainHeading.tsx

```diff
@@ -7,7 +7,7 @@ type MainHeadingProps = {
 
 export function MainHeading({ mainHeading }: MainHeadingProps) {
   return (
-    <Typography sx={{ margin: 0, fontSize: '50px', '@media (min-width: 1280px)': { fontSize: '65px' } }}>
+    <Typography sx={{ margin: 0, fontSize: '50px', fontWeight: 'bold', '@media (min-width: 1280px)': { fontSize: '65px' } }}>
```

Comment:
🔧 I think this would make sense as the `h1` variant in the theme.  Would you define it there and then update this to read from the theme?